server/src/main/java/org/elasticsearch/gateway/LocalAllocateDangledIndices.java

```diff
@@ -105,6 +103,23 @@ public void messageReceived(final AllocateDangledRequest request, final Transpor
                 indexNames[i] = request.indices[i].getIndex().getName();
             }
             final String source = "allocation dangled indices " + Arrays.toString(indexNames);
+
+            var listener = new AllocationActionListener<>(ActionListener.<AllocateDangledResponse>wrap(response -> {
+                try {
+                    channel.sendResponse(new AllocateDangledResponse());
```

Comment:
I think we already have an `AllocateDangledResponse` to hand, no need to make another one?

```suggestion
                    channel.sendResponse(response);
```